projects/components/src/quick-search/quick-search.component.html

```diff
@@ -1,3 +1,11 @@
+<ng-template #sectionTitle let-section>
+    {{ section.provider.sectionName }}
+    <clr-icon *ngIf="section.icon" [attr.shape]="section.icon" size="20"> </clr-icon>
+    <span *ngIf="section.hasPartialResult; let partial" class="partial-result">
```

Comment:
What if some languages require that text to be arranged in a different manner?
Maybe we should have this as part of i18n:
{sectionName} ({lastItem} of {totalItems})